walkthroughs/howto-k8s-appmesh-load-test/fortio.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: fortio
+  namespace: tls-e2e
+spec:
+  podSelector:
+    matchLabels:
+      app: fortio
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+  backends:
+    - virtualService:
+        virtualServiceRef:
+          name: service-0       # Replace with one of the fishap VS
```

Comment:
Is this something customer need to do?

Reply:
To run the load test as it is, they do not. But if they want to customize the tests, they can. For instance, they can replace the service name to send load traffic into different node.